### DISCUSSION TITLE
Modelling diminishing return

### DISCUSSION THREAD

**Discussion by vtkuzmin-farfetch** [sibylhe/mmm_stan#52] [Q&A]
Hi Sibyl He!
Thank you for your realisation and the blog post. 

To model media saturation effect, you fitting a separate model: spend vs impressions. 
What if there is no access to the impressions but only to the spend? Also doesn't it miss saturation effect between impressions and revenue? 
The google paper builds additive model and uses hill function as a transformation. In multiplicative model it's harder, because we use log1p transform. 
What do you think is the best way to incorporate hill function into the main model to have saturation effect between impressions and revenue?
Thank you in advance!

**Comment by sibylhe** [comment]
Impression data is a plus, not necessary. If there is no access to the impressions. the model can be built with spend data only. I think the transformation of spend would be the same as google's paper, followed by log1p, i.e., X = log1p(hill(adstock(spend))), y = log1p(sales)
